libsupport/src/HostAllocator.cpp

```diff
@@ -0,0 +1,16 @@
+#include "katana/HostAllocator.h"
+
+namespace katana {
+
+HostHeap::~HostHeap() {}
+
+SwappableHostHeap::~SwappableHostHeap() {}
+
+SwappableHostHeap swappable_host_heap;
```

Comment:
You can declare this variable as a static variable inside the `GetSwappableHostHeap()` function below. This will ensure correct initialization.